plugins/TodayAtMun/__init__.py

```diff
@@ -41,12 +44,17 @@ async def today_after(self, ctx: commands.Context):
         """Sends the event after the 'next' event."""
         self.today_fun.set_current_date()
         self.today_fun.find_event(self.today_fun.date)
+        self.today_fun.next_day()
         self.today_fun.next_event(self.today_fun.date)
         embed = discord.Embed(
```

Comment:
wanna create a generic function to create an embed?
you're setting the colour + footer to the same thing in both cases, how about having a helper method to create a new embed?

for example, [here](https://github.com/MUNComputerScienceSociety/Automata/blob/master/plugins/MCWhitelist/__init__.py#L44) in the MCWhitelist plugin, I have a generic method for posting an embed based off of an input argument, so I don't have to create it multiple times within the code.

Reply:
I will do that. Good idea